src/ResolvedName.re

```diff
@@ -26,12 +26,16 @@ and emitExportModuleItems = (exportModuleItems, ~buffer) => {
   );
 }
 and emitExportModuleItem = (exportModuleItem, ~buffer) => {
+  let endsWithOpenBracket = Str.regexp(".*{ $");
+
   Buffer.add_string(buffer, "{ ");
   Hashtbl.iter(
     (fieldNameName, exportModuleValue) => {
+      buffer |> Buffer.to_bytes |> Str.string_match(endsWithOpenBracket, _, 0)
+        ? () : Buffer.add_string(buffer, ", ");
```

Comment:
is there a nicer way to do this ?

Reply:
Yes: iterate once to create the list. Then concatenate with String.concat.

Reply:
will do